### ISSUE TITLE
Add support for ephemeralStorage

### ISSUE THREAD

**Issue by deevus** [pulumi/pulumi-awsx#864]
## Hello!

- Vote on this issue by adding a 👍 reaction
- If you want to implement this feature, comment to let us know (we'll work with you on design, scheduling, etc.)

## Issue details

Please add support for the `ephemeralStorage` parameter for `ECSService` and `FargateService`. It is supported in the `aws` package but not `awsx`.

**Comment by stack72** [comment]
Hi @deevus 

So this is actually already available in the new multi-language version of AWSX

It's part of the task definition and can look something like this:

```
const service = new awsx.ecs.FargateService("my-service", {
    cluster: cluster.arn,
    taskDefinitionArgs: {
        ephemeralStorage: {
          sizeInGib: 20,  
        },
        container: {
            image: "nginx:latest",
            cpu: 512,
            memory: 128,
            essential: true,
            portMappings: [
                {
                    containerPort: 80,
                    targetGroup: lb.defaultTargetGroup,
                },
            ],
        },
    },
});
```

Please let me know if there are any issues

Paul

**Comment by deevus** [comment]
Thanks. And how does one use this new version? I’ve already got the
@pulumi/awsx package installed from npm. 

On June 4, 2022, GitHub ***@***.***> wrote:
> Closed #864 <https://github.com/pulumi/pulumi-awsx/issues/864> as
> completed.
>

—
Reply to this email directly, view it on GitHub
<https://github.com/pulumi/pulumi-awsx/issues/864#event-6741658369>, or
unsubscribe <https://github.com/notifications/unsubscribe-
auth/AAGRBARNFD6TLYO3RMPBN2TVNNJWJANCNFSM5XYTZG4Q>.
You are receiving this because you were mentioned.Message ID:
***@***.***>

**Comment by deevus** [comment]
Nevermind... it was the beta version. `yarn` doesn't install beta versions unless you force it.
